capture-sdk/sdk/src/main/res/layout/gc_detection_error_layout.xml

```diff
@@ -0,0 +1,30 @@
+<androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/gc_medium"
+    android:background="@drawable/gc_detection_error_background"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <TextView
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/gc_detection_error_popup_dismiss_button"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toBottomOf="parent"
+        android:textColor="@color/gc_dark_02"
+        android:text="QR code scanning is currently unavailable. Please take a picture of your invoice instead."/>
+
+
+    <TextView
+        android:id="@+id/gc_detection_error_popup_dismiss_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:textColor="@color/gc_accent_01"
+        android:padding="@dimen/gc_large"
+        android:text="Dismiss"/>
```

Comment:
[Request] Please use a string resource and for German use: "Schließen".

Reply:
I will use this one "gc_snackbar_dismiss", because don't want to create new one